examples/compose/docker-compose-with-web.yml

```diff
@@ -0,0 +1,57 @@
+version: '2.1'
+
+services:
+  redis:
+    image: autopilotpattern/redis:3.2.8-r1.0.0
+    mem_limit: 512m
+    restart: always
+    expose:
+      - 6379
+      - 26379
+    environment:
+      - CONSUL=consul
+    links:
+      - consul:consul
+    ports:
```

Comment:
It is easier to experiment with them when open, and for somebody not familiar with Triton, it's kind of important to explain.